qgis-app/plugins/views.py

```diff
@@ -1364,7 +1364,11 @@ def version_approve(request, package_name, version):
         return HttpResponseRedirect(version.get_absolute_url())
     version.approved = True
     version.save()
-    msg = _('The plugin version "%s" is now approved' % version)
+    msg = _(
+        "The plugin version '%s' is now approved. "
+        "Please note that there may be a delay of 10 to 15 minutes "
```

Comment:
or just something like "up to 15 minutes"...

Reply:
Thanks for the review. I've updated it to "up to 15 minutes".